docs/data-sources/logdna_alert.md

```diff
@@ -0,0 +1,66 @@
+# Data Source: `logdna_alert`
+
+Pulls in the relevant details from existing [LogDNA Preset Alerts](https://docs.logdna.com/docs/alerts). The `logdna_alert` _data source_ remotely fetches the information from a preset alert using its ID. This preset alert may or may not be directly managed by Terraform. For the management of preset alerts as Terraform _resources_, refer to the documentation [here](../resources/logdna_alert.md).
+
+Users may opt to reference certain preset alerts as data sources rather than definining them as resources. In this scenario, the preset alerts would not be managed as Terraform _resources_ but the data can still be accessed and referenced in other modules.
+
+To create a `logdna_alert` data source, the `presetid` argument must be provided in its declaration to ensure the data being read remotely is correct.
+
+## Example Usage
+
+```hcl
+provider "logdna" {
+  servicekey = "xxxxxxxxxxxxxxxxxxxxxxxx"
+  url = "https://api.logdna.com" # (Optional) specify a LogDNA region
+}
+
+resource "logdna_alert" "my_alert" {
+  name = "My Preset Alert via Terraform"
+  email_channel {
+    emails          = ["test@logdna.com"]
+    immediate       = "false"
+    operator        = "presence"
+    triggerlimit    = 15
+    triggerinterval = "15m"
+    terminal        = "true"
+    timezone        = "Pacific/Samoa"
+  }
+}
+
+# create data source by referencing the ID from an alert declared in the same config
+data "logdna_alert" "my_alert_remote" {
+  presetid = logdna_alert.my_alert.id
+}
+
+# create data source by pulling in the details from an alert associated with the ID
+data "logdna_alert" "external_remote" {
+  presetid = "xxxxxxxxxx" # the associated ID can be grabbed from the Web UI, API calls, Terraform config, etc
+}
+
+# pass in data source attributes as arguments for module(s) declared in the same config
+resource "logdna_view" "test" {
+  name  = "Basic View"
+  query = "level:debug my query"
+
+  email_channel = data.logdna_alert.my_alert_remote.email_channel
```

Comment:
What is the behavior for attaching preset alerts with integrations not supported via terraform?